go.work

```diff
@@ -0,0 +1,6 @@
+go 1.18
```

Comment:
go.work is a flexibility to test multiple module dependencies during development phase. However the recommendation is to use the tagged version of dependent module in go.mod file. go.work is recommended to place in git.ignore and not recommended for production as well. Other option is to run "go build" command with "GOWORK=OFF" and this requires changes in our dockerfile.

Reply:
The current PR changes are for dev phase only and without go workspace the operator image doesn't get built locally and also on travis as operator now depends on driver module, so to merge this PR in IBM branch - the workspace file needs to be there in the PR, few of the current dependencies are pointing to my fork and I am going to adjust the dependencies after this PR is merged and I have another task already to look at these dependencies - https://github.ibm.com/ibmspectrumscale/csi-tracker/issues/81.